src/patternfly/components/Table/table.scss

```diff
@@ -334,7 +333,6 @@
       top: calc(var(--pf-c-table--BorderWidth) * -1);
```

Comment:
Just noticed a typo at https://github.com/patternfly/patternfly-next/pull/1901/files#diff-cc8438917ce7d4aaec059eff02f314ddR319 Table toggle compount -> compound. Would you mind fixing that?